.gitignore

```diff
@@ -273,3 +273,6 @@ lcov.info
 *.DS_Store
 /WalletWasabi.Backend/wwwroot/onion-index.html
 /NuGet.Config
+
+# vim
+*.swp
```

Comment:
what's this about?

Reply:
Avoid adding `vim` swap files in source code.

Reply:
good, I thought so. [vim for the win!]

Though this is a general change to the repo, not really specific for feeRate in RPC. So, it might be an idea to do an independent PR for it. But well... I think it's a nobrainer to add, so it might be alright. Idk, no issue if others don't care too.

Reply:
Your concept is right Max, in this case, it is OK to make things easier.